tests/jballerina-unit-test/src/test/resources/test-src/error/error_test.bal

```diff
@@ -344,12 +344,12 @@ function bar2(){
     bar();
 }
 
-function testStackOverFlow() returns [runtime:CallStackElement[], string]? {
-    error? e = trap bar();
-    if (e is error){
-        return [e.stackTrace().callStack, e.message()];
-    }
-}
+//function testStackOverFlow() returns [runtime:CallStackElement[], string]? {
```

Comment:
Are we commenting this out because of the `ballerina/runtime` dependency? The error langlib has a `CallStackElement` record atm, so I think we can remove the runtime import and just do

```suggestion
function testStackOverFlow() returns ['error:CallStackElement[], string]? {
```

or even 

```suggestion
function testStackOverFlow() returns [any, string]? {
```

and enable this test also.